src/Interpreters/Cache/FileCache.cpp

```diff
@@ -141,7 +143,16 @@ bool FileCache::isInitialized() const
         /// Exception during initialization,
         /// make user aware that the cache is broken.
         if (init_exception)
-            std::rethrow_exception(init_exception);
+        {
+            if (throw_on_cache_initialization_error)
+            {
+                std::rethrow_exception(init_exception);
+            }
+            else
+            {
+                tryLogCurrentException(log, "Cache failed to initialize: ");
```

Comment:
No need to log it here, it should be logged once - when the init itself failed, otherwise it will pollute the logs.

Reply:
Removed this log as it is already logged in `initializeImpl()`